gettsim/policy_for_date.py

```diff
@@ -55,9 +56,11 @@ def get_policies_for_date(year, group, month=1, day=1, raw_group_data=None):
         else:
             tax_data["calc_hhfreib"] = calc_hhfreib_from2015
         if year >= 2010:
-            tax_data["vorsorge"] = vorsorge2010
-        else:
-            tax_data["vorsorge"] = vorsorge_dummy
+            tax_data["vorsorge"] = vorsorge04_10
+        elif year >= 2005:
+            tax_data["vorsorge"] = vorsorge04_05
+        elif year <= 2004:
+            tax_data["vorsorge"] = vorsorge2004
```

Comment:
No test case for this, if I interpret the coverage statistics correctly?

Reply:
Test is added. I am struggling with a correct implementation though. The test wants to compare the column "vorsorge", so I need to return a named item. This however leads to problems if `_vorsorge_pre_2005()` returns only an `int`. Any ideas, @hmgaudecker , @tobiasraabe ?

Reply:
I do not quite understand. In my world, the column "vorsorge" should already exist in the DataFrame (`tax_data` in the test) and it should get modified in-place by the functions? And all of them should share a common interface.

Reply:
working now...